webviz_config/_docs/_create_schema.py

```diff
@@ -8,35 +8,126 @@
 JSON_SCHEMA = {
     "$id": "https://github.com/equinor/webviz-config",
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "Person",
+    "title": "Webviz",
     "type": "object",
     "properties": {
         "title": {"description": "Title of your Webviz application.", "type": "string"},
         "shared_settings": {"type": "object"},
-        "pages": {
-            "description": "Define the pages in your Webviz application.",
-            "type": "array",
-            "minLength": 1,
-            "items": {
```

Comment:
Technically it is `oneOf` `pages` or `layout` in this transition phase?

Reply:
Isn't the schema just for helping to create new config files and shouldn't these new files rather follow the new schema? Or is it also used for validating config files?

Reply:
It is currently used for both. When people open an existing file in VS Code and do modifications, they get continuous feedback regarding if the new changes make the file invalid.

Not very important, we could probably live with existing files being opened and be marked as invalid by the VS Code linter. What do you think @rubenthoms @asnyv @HansKallekleiv?